iroha-lib/examples/DomainAssetCreation.cpp

```diff
@@ -0,0 +1,251 @@
+#include <iostream>
+#include <string>
+#include <cassert>
+
+#include "model/Tx.hpp"
+#include "model/Query.hpp"
+#include "model/utils/Utils.h"
+
+
+constexpr const char* LINE = "-----------";
```

Comment:
```suggestion
static constexpr const char* LINE = "-----------";
```

Reply:
```suggestion
static constexpr std::string LINE = "-----------";
```

Reply:
Here it can't be `constexpr string`. After changing into `fmt::print` I decided that more readable is not to have the const:
```
fmt::print("----------->{}-----------\n", __FUNCTION__);
```
So I mark as resolved